packages/button/src/Button.tsx

```diff
@@ -238,14 +238,18 @@ const baseStyle = css`
   transition: all 120ms ease;
   text-decoration: none;
   cursor: pointer;
-  &:disabled {
-    color: ${colors.gray[3]};
-    border-color: ${colors.gray[5]};
-    background-color: ${colors.gray[7]};
-    background-image: none;
-    box-shadow: none;
-    cursor: not-allowed;
-  }
+`;
+
+const disabledStyle = css`
+  color: ${colors.gray[3]};
+  border-color: ${colors.gray[5]};
+  background-color: ${colors.gray[7]};
+  background-image: none;
+  box-shadow: none;
+`;
+
+const disabledAnchor = css`
+  pointer-events: none;
```

Comment:
Is there any reason this rule shouldn't be a part of `disabledStyle`? I don't see a reason to not have this be a part of both the button and anchor tag styles.

Reply:
I don't know to be honest, felt unnecessary for other cases but I'll merge them

Reply:
👍

Reply:
done